packages/broker/src/plugins/brubeckMiner/BrubeckMinerPlugin.ts

```diff
@@ -124,6 +127,7 @@ export class BrubeckMinerPlugin extends Plugin<BrubeckMinerPluginConfig> {
             clientServerLatency: this.latestLatency,
             waitTime: delay,
             natType: this.natType,
+            beneficiaryAddress: this.beneficiaryAddress,
```

Comment:
We could read the info from `this.pluginConfig.beneficiaryAddress` and remove the `this.beneficiaryAddress` field?